commands/build.go

```diff
@@ -643,6 +652,101 @@ func parsePrintFunc(str string) (*build.PrintFunc, error) {
 	return f, nil
 }
 
+func addGitProvenance(ctx context.Context, labels []string, contextPath string, dockerfilePath string) ([]string, error) {
```

Comment:
nit: I would rather if we could avoid taking `labels` as an argument here (since we don't modify any of the contents, except for adding more), and append it at the call site.

Reply:
Done